e2e/meta_test/test_schema_nodes_properties/test.yml

```diff
@@ -0,0 +1,19 @@
+query: >
+    CALL meta.schema(true) YIELD nodes, relationships UNWIND nodes AS node RETURN node.properties.count AS count, node.properties.properties_count.age AS age, node.properties.properties_count.name AS name, node.properties.properties_count.year AS year;
+output:
+  - count: 1
+    age: 1
+    name: 1
+    year: 
+  - count: 2
+    age: 2
+    name: 2
+    year: 1
+  - count: 2
+    age: 
+    name: 2
+    year:
+  - count: 1
+    age: 
+    name: 1
+    year: 
```

Comment:
I am not sure if it is possible for this to break, meaning we have "no guarantee" of order, right? Is it possible to somehow guarantee it?

Reply:
I am also not sure, but I also thought about that before, so I think it makes sense to add order to be sure. Gonna do that.